Content.Server/Weapons/Ranged/Systems/GunSystem.cs

```diff
@@ -303,6 +318,14 @@ private void OnBallisticPrice(EntityUid uid, BallisticAmmoProviderComponent comp
 
     private void ShootOrThrow(EntityUid uid, Vector2 mapDirection, Vector2 gunVelocity, GunComponent gun, EntityUid gunUid, EntityUid? user)
     {
+        if (gun.Target != null && TryComp<FixturesComponent>(uid, out var fixtures))
+        {
+            var fixture = fixtures.Fixtures.First();
+            Physics.SetCollisionMask(uid, fixture.Key, fixture.Value, (int) CollisionGroup.LayingDownMobMask);
```

Comment:
You can't index dictionaries like this.

Reply:
I changed it so it goes through all the fixtures, might not be the best solution though since I am changing all the fixtures masks.